docs/querying-data.rst

```diff
@@ -0,0 +1,207 @@
+Querying data in Kingfisher Views
+=================================
+
+Summary tables in Kingfisher Views are generated from data in `Kingfisher Process <https://kingfisher-process.readthedocs.io/en/latest/>`__, which is organized by collection. Each collection contains either data downloaded from an OCDS publisher using a `Kingfisher Scrape <https://kingfisher-scrape.readthedocs.io/en/latest/>`__ spider, or loaded locally.
+
+A Kingfisher Views schema can contain summaries of one or more collections.
+
+OCDS data can be published using either `releases or records <https://standard.open-contracting.org/latest/en/getting_started/releases_and_records/>`__. Releases are point in time updates about a contracting process and a record provides an index of all the releases for a contracting process.
+
+Kingfisher Process automatically transforms OCDS releases to create compiled release collections,  containing a single compiled release per contracting process populated with the latest value of each field.
+
+Tables in Kingfisher Views can contain summaries of:
+
+* individual releases
+* compiled releases generated by Kingfisher Process
+* compiled releases published as part of an OCDS record
+
+Since most analysis is much easier to perform on compiled releases, we recommend working with compiled releases to begin with.
+
+List all schemas in the database
+--------------------------------
+
+Kingfisher Views creates database schemes to store summary data.
+
+The following query returns a list of schemas in the database:
+
+.. code-block:: sql
+
+  select
+    schema_name
+  from
+    information_schema.schemata;
+
+Kingfisher Views schemas are named ``view_data_*``, if you don't see any such schemas refer to :doc:`cli/index` and use the ``add-view`` command to generate a schema.
+
+Set the schema to query
+-----------------------
+
+The following command sets the current schema to ``view_data_collection_1070_1071_1072`` by making it the first item in the search path:
+
+.. code-block:: sql
+
+  set
+    search_path
+  to
+    view_data_collection_1070_1071_1072, public;
+
+.. note::
+
+  Depending on the tool used to query the database, you may need to include the above command at the start of each query you run.
+
+List the collections in the current schema
+------------------------------------------
```

Comment:
`selected_collections` doesn't have the release type, so I prefer to do it this way so the user can easily pick a compiled release collection to work with.

Reply:
But can't you join selected_collections to collection and use the cache counts to determine the types available? select distinct on release_summary can be very slow in comparison.

Reply:
I can update it if you strongly prefer that approach, but the user only needs to run the query once to identify the collection to work with and the current approach takes less than 4 seconds to run on schema which has more than 9 million entries in `release_summary`.

I chose this approach because I think the output is easier to understand for a new user.

Reply:
Fine to keep. Output is indeed easier to interpret.